### ISSUE TITLE
change disk space used from df to api/sno

### ISSUE THREAD

**Issue by bjoerrrn** [bjoerrrn/storj-system-health.sh#30]
`/api/sno/`

```
"diskSpace":{
      "used":1374306238338,
      "available":9500000000000,
      "trash":58233200156,
      "overused":0
   },
```

**Comment by dusselmann** [comment]
.. and add warning, if difference to real disk space is loo large and might cause issues, when the disk gets full sometimes: 

[full disk space - allocated disk space] < [current real disk space - sno disk space]

**Comment by bjoerrrn** [comment]
fixed with version 1.6.7
